db/migrate/20211107024057_change_developers_available_on.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeDevelopersAvailableOn < ActiveRecord::Migration[7.0]
+  def change
+    change_column_null :developers, :available_on, true
```

Comment:
Oh! I didn't know this was a thing. TIL.

Reply:
I just found out too 

API https://edgeapi.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-change_column_null

Reply:
I've always done `change_column :developers, :available_on, null: false` but you have to remember to copy over any other params, like `default:`. Yours is so much better!